### PR TITLE
Genres and few more

### PR DESCRIPTION
**Pull request by lkkolosowski**
I Implemented fetching genres from API, and modified a few things to match the current changes😊 I hope you guys like it 😉

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ function App() {
           <PeopleList />
         </Route>
         <Route path="/">
-          <MoviesList />
```
Comment (Albrecht-Albi) [inline src/App.js:18]: extra we already have a router for movies :)

File: src/common/Header/styled.js [hunk 2]
```diff
@@ -94,10 +94,12 @@ export const Loupe = styled(LoupeIcon)`
   left: 24px;
   top: 50%;
   transform: translateY(-50%);
```
Comment (Albrecht-Albi) [inline src/common/Header/styled.js:96]: Good idea with the padding correction, now they will smooth out better

File: src/common/Tiles/Info/index.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { Title, Subtitle } from "./styled";
+import { extractFirstWord } from "./utils";
+
+export const MainInfo = ({ title, year }) => (
+  <div>
+    <Title>{title}</Title>
+    <Subtitle>{extractFirstWord(year)}</Subtitle>
+  </div>
+);
```
Comment (Albrecht-Albi) [inline src/common/Tiles/Info/index.js:9]: Nice simple code you used :)

File: src/common/Tiles/Info/styled.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  max-height: 2.6em;
+  color: ${({ theme }) => theme.color.woodsmoke};
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.large}) {
+    font-size: 16px;
+    margin: 0;
+    max-height: 5.2em;
+    -webkit-line-clamp: 4;
+  }
+`;
```
Comment (Albrecht-Albi) [inline src/common/Tiles/Info/styled.js:22]: You've captured the design perfectly. You seem to be pretty good

File: src/common/Tiles/Rating/styled.js [hunk 5]
```diff
@@ -8,25 +8,28 @@ export const Wrapper = styled.div`
...
 `;
 
 export const Star = styled(StarIcon)`
-width: 24px;
-margin-top: -6px;
+  width: 24px;
+  margin-top: -6px;
 
   @media (max-width: ${({ theme }) => theme.breakpoint.large}) {
     width: 16px;
```
Comment (Albrecht-Albi) [inline src/common/Tiles/Rating/styled.js:35]: Good formatting, I thought you forgot

File: src/features/getDataFromAPI.js [hunk 6]
```diff
@@ -1,12 +1,26 @@
...
 
+const URL = "https://api.themoviedb.org/3";
+const API_KEY = "1a89c6b78383e65e62937e5f4ffe7915";
+
 export const getPopularMovies = async () => {
   const response = await axios.get(
-    "/movies-browser/exampleDataAPI.json"
+    `${URL}/movie/popular?api_key=${API_KEY}&language=en-US&page=1`
   );
   if (!response.ok) {
```
Comment (Albrecht-Albi) [inline src/features/getDataFromAPI.js:10]: you've done a great job with the API, department ;)

File: src/features/moviesListSlice.js [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+export const { fetchMoviesListSuccess, fetchMoviesListError } =
+  moviesListSlice.actions;
+
+export const selectMoviesListState = (state) => state.moviesList;
+export const selectMoviesList = (state) =>
+  selectMoviesListState(state).movies;
+export const selectGenres = (state) => selectMoviesListState(state).genres;
+export const selectStatus = (state) => selectMoviesListState(state).status;
+
+export default moviesListSlice.reducer;
```
Comment (Albrecht-Albi) [inline src/features/moviesListSlice.js:31]: nice code you did, you are doing a great job encoding the file.

**Review by Albrecht-Albi — APPROVED** [review]

**Review by Albrecht-Albi — APPROVED** [review]

**Comment by lkkolosowski** [comment]
Thanks for approve 😊

**Pull request merged**